sources/api/pluto/src/main.rs

```diff
@@ -270,11 +271,68 @@ fn pause_container_uri(region: &str) -> Result<String> {
     ))
 }
 
+/// Calculates the amount of memory to reserve for kubeReserved in mebibytes.
+/// This function is taken from:
+/// https://github.com/awslabs/amazon-eks-ami/blob/db28da15d2b696bc08ac3aacc9675694f4a69933/files/bootstrap.sh#L228-L239
+fn get_memory_mebibytes_to_reserve(client: &Client, session_token: &str) -> Result<String> {
+    let max_num_pods = get_max_pods(&client, &session_token)?;
+    let mebibytes_unit = "Mi";
+    let max_num_pods_u32 = max_num_pods.parse::<u32>().context(error::ParseToU32 { object: max_num_pods })?;
```

Comment:
Can you try running `cargo fmt` on pluto. I think this line and several other places in this file needs formatting.

Reply:
sure thing